src/_posts/security/cryptography/2000-01-01-tls.md

```diff
@@ -0,0 +1,39 @@
+---
+title: TLS Version
+modified_at: 2023-08-29 00:00:00
+tags: security cryptography tls ciphers
+---
+
+## Definition
+
+Transport Layer Security (TLS) is a protocol that ensures the confidentiality of exchanges between applications and users on the Internet. When a server and client communicate, TLS ensures that no third party can intercept or falsify a message. TLS is the successor to the SSL (Secure Sockets Layer) protocol.
+
+This page aims at listing compatible TLS versions with the Scalingo platform.
+
+## Scopes
+
+Front servers which handle all incoming HTTP/HTTPS requests (see [reverse proxy](https://doc.scalingo.com/platform/internals/network#reverse-proxy))
```

Comment:
That's not how you make link in the documentation. Such link does not work for review apps or in dev.
```suggestion
Front servers which handle all incoming HTTP/HTTPS requests (see [reverse proxy]({% post_url platform/internals/2000-01-01-network %}#reverse-proxy))
```